flexmeasures/auth/__init__.py

```diff
@@ -31,6 +32,11 @@ def register_at(app: Flask):
     user_datastore = SQLAlchemySessionUserDatastore(db.session, User, Role)
     app.security = Security(app, user_datastore)
 
+    # Global context processor for all templates
```

Comment:
I would write "make sure we use our colors in all templates".

Also, [the context processor is a Flask feature](https://flask.palletsprojects.com/en/2.3.x/templating/#context-processors) and not about security. 
So a better place to do this might be `app.py`, or we could even do it in `ui/__init__.py`, within the `register_at()` function. Then, the  `color_defaults.py` can stay in the place best suited for it - the `ui` package.

Reply:
I mentioned security before because I initially thought this issue only occurred with templates managed by the Flask-Security package, but when @Flix6x pointed out the new place he found the bug I thought I make it global as adding it to the security context didn't cover for the task page which also had the bug.

but I am going to try what you just suggested and revert.

Reply:
Yeah I understand why you placed it here in the first place, then started using the global context, I basically only suggest a better place (in the `ui` blueprint). I'm pretty sure that is the final puzzle piece for this :)

Reply:
yeah, I completely agree with you, I am already working on relocating it to make sure all works fine, and I will make a commit soon.